services/eventservice/src/types.rs

```diff
@@ -0,0 +1,69 @@
+use chrono;
+use serde::{Deserialize, Serialize};
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct Category {
```

Comment:
do you expect other fields to extend Category structure? 
why don't u use just a string?

Reply:
I think category will be string only, but category is not string. Think about it like about typedef. I will change it a bit to be more like typedef